src/main/java/org/jenkinsci/plugins/build_view_column/BuildViewColumnDescriptor.java

```diff
@@ -16,6 +16,10 @@ protected BuildViewColumnDescriptor() {
     }
 
     public static DescriptorExtensionList<BuildViewColumn,BuildViewColumnDescriptor> all() {
-        return Jenkins.getInstance().getDescriptorList(BuildViewColumn.class);
+        try{
+            return Jenkins.getInstance().getDescriptorList(BuildViewColumn.class);
+        } catch(NullPointerException ex){
```

Comment:
I like ExtensionList.lookup because it won't throw an exception.  However, the return type is different so I'm going with the path of least resistance.